frontend/src/pages/projects/screens/detail/notebooks/NotebookStorageBars.tsx

```diff
@@ -39,40 +46,51 @@ const NotebookStorageBars: React.FC<NotebookStorageBarsProps> = ({ notebook, onA
           </Alert>
         ) : (
           <List isPlain>
-            {pvcs.map((pvc) => (
+            {visiblePvcs.map((pvc) => (
               <ListItem key={pvc.metadata.name}>
-                <Stack>
-                  <StackItem>
+                <Flex
+                  gap={{ default: 'gapSm' }}
+                  alignItems={{ default: 'alignItemsFlexStart' }}
+                  flexWrap={{ default: 'nowrap' }}
+                >
+                  <FlexItem>
+                    <HeaderIcon display="block" type={ProjectObjectType.clusterStorage} size={20} />
+                  </FlexItem>
+                  <FlexItem>
                     <Text component="small">{getDisplayNameFromK8sResource(pvc)}</Text>
-                  </StackItem>
-                  <StackItem>
-                    <StorageSizeBar pvc={pvc} />
-                  </StackItem>
-                  <StackItem>
-                    <Text component="small">
-                      Mount path:{' '}
-                      {mountFolderMap[pvc.metadata.name]
-                        ? mountFolderMap[pvc.metadata.name]
-                        : 'Unknown'}
-                    </Text>
-                  </StackItem>
-                </Stack>
+                    <Stack>
+                      <StackItem>
+                        <StorageSizeBar pvc={pvc} />
+                      </StackItem>
+                      <StackItem>
+                        <Text component="small">
```

Comment:
these texts should not be small anymore, the mocks show them as the same size as the others